src/libasr/ASR.asdl

```diff
@@ -265,6 +265,7 @@ expr
     | ListSection(expr a, array_index section, ttype type, expr? value)
     | ListPop(symbol a, expr? index, ttype type, expr? value)
     | DictPop(symbol a, expr key, ttype type, expr? value)
+    | SetPop(symbol a, ttype type, expr? value)
```

Comment:
This needs to be:
```suggestion
    | SetPop(expr a, ttype type, expr? value)
```

Reply:
Yes, they all have to be changed to `expr`.

Reply:
> I think the nodes in 267 and 266 also need a similar change. `ListPop(expr a, ...)`, `DictPop(expr a, ...)`

Please see: https://github.com/lcompilers/lpython/pull/500#discussion_r873772682.